### ISSUE TITLE
Performance comparison with another popular RL library

### ISSUE THREAD

**Issue by LinxiFan** [pfnet/pfrl#92]
Thank you so much for open-sourcing this wonderful library. I wonder how does PFRL compare against this popular PyTorch RL library: https://github.com/ikostrikov/pytorch-a2c-ppo-acktr-gail

From the learning curves of PPO on Gym environments, it looks like that repo has much higher scores? Here is their half-cheetah curve, which reaches ~3000 score at only 1M steps. 
![image](https://user-images.githubusercontent.com/3018029/98884909-e8158580-2445-11eb-95c4-54b41b4a662f.png)

VS PFRL's curve:

![image](https://user-images.githubusercontent.com/3018029/98884955-febbdc80-2445-11eb-9334-cc6bb75fa425.png)

**Comment by ummavi** [comment]
Hello, and thank you for the kind words! 

As mentioned in our [PPO readme](https://github.com/pfnet/pfrl/tree/master/examples/mujoco/reproduction/ppo), our reproduction script focuses on the reproduction of the settings and performance from the [Deep Reinforcement Learning that Matters](https://arxiv.org/abs/1709.06560) paper and do match it quite closely as shown below:
![Screenshot 2020-11-18 at 12 10 12](https://user-images.githubusercontent.com/6052050/99485914-b735d300-29a6-11eb-9be3-07d7723c9b11.png)

We note that this is in no way claimed as the optimal hyper-parameters for the environment and any number of implementation or parameter changes could make a world of a difference, but we tried to be faithful to the paper as possible so the script could potentially be used as a benchmark in research.

I'm not very familiar with the library you've linked, but a quick glance at the runscripts and parameters from the README let me notice a few differences: 
(assuming they run the script with the parameters `python main.py --env-name "Reacher-v2" --algo ppo --use-gae --log-interval 1 --num-steps 2048 --num-processes 1 --lr 3e-4 --entropy-coef 0 --value-loss-coef 0.5 --ppo-epoch 10 --num-mini-batch 32 --gamma 0.99 --gae-lambda 0.95 --num-env-steps 1000000 --use-linear-lr-decay --use-proper-time-limits`) 

Gamma (0.99 vs 0.995)
Lambda (0.95 vs 0.97)
Batch Size (32 vs 64)
Learning rate schedule (Linear Decay vs Constant) 
Gradient Norm clipping (0.5 vs None)
The use of "proper" time limits vs None
Observation Normalization (None vs by running mean)

Since RL's [notoriously sensitive even to the smallest of changes](https://arxiv.org/abs/1709.06560), I'm certain a combination of these and small implementation changes (such as the normalization coefficients for advantage standardizations- (1e-5 vs 1e-8)) are the reason for those differences and effort towards matching them should hopefully get them closer to each other)

**Comment by prabhatnagarajan** [comment]
Closing this issue due to inactivity.
